cmd/buildctl/build/output.go

```diff
@@ -46,9 +46,6 @@ func parseOutputCSV(s string) (client.ExportEntry, error) {
 	if err != nil {
 		return ex, errors.Wrap(err, "invalid output option: output")
 	}
-	if ex.Output != nil || ex.OutputDir != "" {
```

Comment:
keeping `ex.Attrs["dest"]` here because it is important in `deleteExportedFiles()`

Reply:
I prefer deleting the attr and returning value to the caller

Reply:
@AkihiroSuda 
What about adds a `Dest  string` to `type ExportEntry struct{}` to preserve this value? 🤔